cmd/juju/status/status_internal_test.go

```diff
@@ -94,19 +94,19 @@ func test(summary string, steps ...stepper) testCase {
 }
 
 type stepper interface {
-	step(c *gc.C, ctx *context)
+	step(c *gc.C, ctx *ctx)
 }
 
 //
-// context
+// ctx
 //
 
 type charmInfo struct {
 	charm charm.Charm
 	url   string
 }
 
-type context struct {
+type ctx struct {
```

Comment:
`testContext` or `fakeContext` or something might be good too.

Reply:
Yes good idea